src/shared/dissect-image.c

```diff
@@ -150,7 +150,8 @@ static int device_is_partition(sd_device *d, blkid_partition pp) {
                 return false;
 
         r = sd_device_get_sysattr_value(d, "partition", &v);
-        if (r == -ENOENT) /* Not a partition device */
+        if (r == -ENOENT ||        /* Not a partition device */
+            ERRNO_IS_PRIVILEGE(r)) /* Not ready to access? */
```

Comment:
@bluca Could you check this?

Reply:
Hmm, why would we see EPERM/EACCESS here? the "partition" sysfs attribute is not accessible initially? that's so strange

Reply:
Yeah, but it seems `lstat()` return EPERM.

Reply:
This is so strange... The old code hid that error code I guess. But I wonder how this happens?

Reply:
sorry - no clue, it seems very strange

Reply:
#18717 hides the error again :-p